README.md

```diff
@@ -68,15 +75,19 @@ pub fn SomeComponent() -> impl IntoView {
 
 ### Scoped Hotkeys
 
+> [!TIP]
+>
 > Assign hotkeys specific to individual sections without collisions using scopes.<br/>
 > Use functions in `HotkeysContext` for scope management.
->
-> This example shows an inner and outer scope and hotkeys that switch between the scopes.
+
+This example shows an inner and outer scope and hotkeys that switch between the scopes.
+
+> [!TIP]
```

Comment:
Could you remove the tip badge? That way, we'll just have one tip block :)

Reply:
What do you think about this version, with a single, unified tip?

https://github.com/gaucho-labs/leptos-hotkeys/blob/6ba60cadddaf6beea7c658a50bce915bd0cb1983/README.md#scoped-hotkeys

The _"This example ..."_ prose text is moved up towards the heading, like it already was in https://github.com/gaucho-labs/leptos-hotkeys/blob/6ba60cadddaf6beea7c658a50bce915bd0cb1983/README.md#global-hotkeys

Reply:
I like the single unified tip :) Thanks. Fwiw, there will be a couple of breaking changes that change how this library works. If you want to join as a collaborator and help out with the readme then, let me know, happy to add you to the repository! 

For now, we should merge this, I think this is great

Reply:
I’m just getting started with Leptos, so contributor access to its ecosystem does not feel appropriate just yet.
But I might come back with issues and PRs eventually - be it code or docs.

In any case, thanks for the offer, and thank you very much for open sourcing your work!

Reply:
Please do come back! Thanks for your help :)